tests/src/Kernel/Entity/GetGroupContentTest.php

```diff
@@ -82,6 +95,7 @@ public function testBasicGroupReferences() {
     $groups['entity_test'] = EntityTest::create([
       'type' => $bundle,
       'name' => $this->randomString(),
+      'uid' => $this->groupAdmin->id(),
```

Comment:
This was only failing before due to the code in og_entity_insert. So maybe
have a look there? Or the test is not really using the entity test entities
as groups?
On 13 Jun 2016 17:01, "Pieter Frenssen" <notifications@github.com> wrote:

> In tests/src/Kernel/Entity/GetGroupContentTest.php
> <https://github.com/amitaibu/og/pull/233#discussion_r66818911>:
>
> > @@ -82,6 +95,7 @@ public function testBasicGroupReferences() {
> >      $groups['entity_test'] = EntityTest::create([
> >        'type' => $bundle,
> >        'name' => $this->randomString(),
> > +      'uid' => $this->groupAdmin->id(),
>
> Getting back to this, in EntityTest the user ID field is actually called
> user_id instead of uid, so if this is required to make the tests pass
> then we probably have a bug somewhere.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amitaibu/og/pull/233/files/65053e383705a9da5b932bbf64772e9271f14599#r66818911>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABAUw-4yKoL4RdVUWDKAa3efpFamRYnsks5qLX66gaJpZM4I0F6I>
> .
>